pkg/cmd/roachtest/tests/mixed_version_backup.go

```diff
@@ -2647,6 +2664,12 @@ func registerBackupMixedVersion(r registry.Registry) {
 				// attempted.
 				mixedversion.UpgradeTimeout(30*time.Minute),
 				mixedversion.AlwaysUseLatestPredecessors,
+				// This test sometimes flake on separate-process
```

Comment:
Nit: I'm wondering if we should have a meta issue / or issues that lists these tests, that are incompatible for unknown reasons, so we don't lose track?

Reply:
Created #130968.